src/test/java/uk/gov/companieshouse/acsp/manage/users/controller/UserAcspMembershipTest.java

```diff
@@ -30,12 +37,11 @@
 @Tag("unit-test")
 class UserAcspMembershipTest {
 
+  @MockBean
+  AcspMembershipService acspMembershipService;
   @Autowired private MockMvc mockMvc;
```

Comment:
Nitpick, but probably best to keep consistent:

```suggestion
  @Autowired private MockMvc mockMvc;

  @MockBean private AcspMembershipService acspMembershipService;
```